specs/agents/metadata.md

```diff
@@ -37,10 +37,13 @@ if os == windows
   hostname = exec "cmd /c hostname"                   // or any equivalent *
   if (hostname == null || hostname.length == 0)
     hostname = env.get("COMPUTERNAME")
+  domain = env.get("USERDNSDOMAIN")
+  if (domain != null && domain.length >0)
+    hostname += "." + domain
 else 
   hostname = exec "uname -n"                          // or any equivalent *
```

Comment:
I think this preference for using `uname -n` first should be dropped now, because it will prevent using `hostname -f` to get the FQDN on Linux.  On BSDs (like macOS) `hostname` and `uname -n` have the same effective source and always return the FQDN. On Linux, my understanding is that `uname -n` just returns the `gethostname` syscall value, which does *not* include the FQDN; where as at least some Linux implementations of `hostname -f` do get the FQDN ([Debian's source](https://sources.debian.org/src/hostname/3.23/hostname.c/#L336-L338)).

Reply:
Yes, that's a very good point, I've removed the `uname -n` heuristic as it's no longer relevant.